### ISSUE TITLE
Incorrect date_order XPaths in IATI Standard Rulesets

### ISSUE THREAD

**Issue by dalepotter** [IATI/IATI-Rulesets#31]
The [IATI ruleset specification](http://iatistandard.org/104/rulesets/ruleset-spec/
) states that, for the `date_order ` rule:

> The date matched by less must be less than the date matched by more. If either of these dates is not found, the rule is ignored.
 
This `date_order ` rule is implemented in the IATI Standard ruleset [here](https://github.com/IATI/IATI-Rulesets/blob/version-2.02/rulesets/standard.json#L11-L14) (same rule at versions 1.04 to 2.02):

```
        "date_order": {
            "cases": [
                { "less": "activity-date[@type='1']", "more": "activity-date[@type='3']" },
                { "less": "activity-date[@type='2']", "more": "activity-date[@type='4']" },
                { "less": "activity-date[@type='2']", "more": "NOW" },
                { "less": "activity-date[@type='4']", "more": "NOW" }
            ]
        },
```
However, in all cases the stated XPaths for `less` and `more` are incorrect, since these are do not relate to that path where the dates are entered. 

To fix, each instance should be suffixed by the `@iso-date` attribute.

This issue appears to applies to all other instances where dates are referred to, i.e. these rules in the [v2.02 ruleset](https://github.com/IATI/IATI-Rulesets/blob/version-2.02/rulesets/standard.json#L45-L108).

**Comment by hayfield** [comment]
This has been fixed in #32, #33, #34, #35

**Comment by andylolz** [comment]
While standard.json was updated for all versions, the relevant code that actually runs the tests wasn’t updated. So this change breaks that.
